### DISCUSSION TITLE
Show the commits that introduced/changed a file

### DISCUSSION THREAD

**Discussion by titanve** [community/community#31511] [Other Feature Feedback, Questions, & Ideas]
Hi!

I think that a good addition to the `Files changed` tab is to have over the file frame what commits changed or introduced the file (changes) so we can quickly copy (cherry-pick) those commits into a new PR. It could be placed below the filename, maybe?

Thanks!
![image](https://user-images.githubusercontent.com/4803351/187728981-2079e1d5-3409-473e-869d-7d6c76bc8144.png)

**Comment by Akash1134** [comment]
Hi @titanve Thanks for sharing this suggestion. I have labelled your post with the relevant tag so the product team can review the request.
